src/test/java/org/tests/order/OrderReferencedParent.java

```diff
@@ -0,0 +1,43 @@
+package org.tests.order;
+
+import io.ebean.annotation.Index;
+
+import javax.persistence.DiscriminatorColumn;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Inheritance;
+import javax.persistence.InheritanceType;
+import javax.persistence.MappedSuperclass;
+
+@Entity
+@MappedSuperclass
```

Comment:
I don't _think_ @MappedSuperclass is strictly necessary here ... could check that after merge.